com.netease.CloudMusic.json

```diff
@@ -54,7 +54,23 @@
             }
         ]
     },
-    
+
+    {
+        "name": "keyutils",
```

Comment:
Is this required? If so, please use https://git.kernel.org/pub/scm/linux/kernel/git/dhowells/keyutils.git as source and if it works bump the version to 1.6.3. Also should probably use LIBDIR instead of copying and cleanup the rest.